flink-python/setup.py

```diff
@@ -321,12 +321,11 @@ def find_file_path(pattern):
         license='https://www.apache.org/licenses/LICENSE-2.0',
         author='Apache Software Foundation',
         author_email='dev@flink.apache.org',
-        python_requires='>=3.5',
-        install_requires=['py4j==0.10.8.1', 'python-dateutil==2.8.0', 'apache-beam==2.23.0',
+        python_requires='>=3.6',
+        install_requires=['py4j==0.10.8.1', 'python-dateutil==2.8.0', 'apache-beam==2.27.0',
                           'cloudpickle==1.2.2', 'avro-python3>=1.8.1,<=1.9.1', 'jsonpickle==1.2',
-                          'pandas>=0.24.2,<1; python_full_version < "3.5.3"',
-                          'pandas>=0.25.2,<1; python_full_version >= "3.5.3"',
-                          'pyarrow>=0.15.1,<0.18.0', 'pytz>=2018.3', 'numpy>=1.14.3,<1.20'],
+                          'pandas>=0.25.2,<1', 'pyarrow>=0.15.1,<0.18.0', 'pytz>=2018.3',
```

Comment:
```suggestion
                          'pandas>=1.0,<1.3.0', 'pyarrow>=0.15.1,<4.0.0', 'pytz>=2018.3',
```

Reply:
I think 'pandas>=1.0,<1.2.0', 'pyarrow>=0.15.1,<3.0.0' is better, which is consistent with the dependency version of `apache-beam`